diesel_cli/tests/print_schema/print_schema_except_tables/diesel.toml

```diff
@@ -1,4 +1,4 @@
 [print_schema]
 file = "src/schema.rs"
 with_docs = true
-filter = { whitelist = ["users1"] }
+filter = { except-tables = ["users1"] }
```

Comment:
This is backwards, isn't it? Shouldn't this be `only-tables`?

Reply:
I think git is just confused.

Reply:
I might have confused the git rename detection?